src/reportSources/legacyFflogs/eventAdapter/deduplicateAoE.ts

```diff
@@ -37,7 +38,16 @@ export class DeduplicateAoEStep extends AdapterStep {
 		}
 
 		// Events with no sequence ID are from over time effects or the passive regeneration, do not deduplicate
-		return this.deduplicatedEvents.find(e => e.type === event.type && e.sequence != null && e.sequence === event.sequence)
-	}
+		if (event.sequence == null) {
+			return undefined
+		}
 
+		const key = `${event.type}:${event.sequence}`
```

Comment:
Minor style nit, we usually include an extra line break after constants and variables.
```suggestion
		const key = `${event.type}:${event.sequence}`
		
```